pageGame.js

```diff
@@ -0,0 +1,20 @@
+import { Selector } from 'testcafe';
+
+export default class Game {
+    constructor() {
+        this.game = Selector('.game_purchase_action_bg');
+        this.realName = Selector('#appHubAppName');
+        this.replaceLetters = /[ a-zA-Z ]/g;
+    }
+
+    async getSecondGame() {
```

Comment:
why do these function have `Second` in their names?
and instead of having two separate functions for game price and game name you can have a single function which will return game object with all fields you need. and same goes for `GenrePage` class